docs/csharp/language-reference/compiler-messages/cs1997.md

```diff
@@ -0,0 +1,47 @@
+---
+description: "Compiler Error CS1997"
+title: "Compiler Error CS1997"
+ms.date: 9/12/2022
+f1_keywords:
+  - "CS1997"
+helpviewer_keywords:
+  - "CS1997"
+---
+# Compiler Error CS1997
+
+Since is an async method that returns 'Task', a return keyword must not be followed by an object expression. Did you intend to return 'Task\<T\>'?
```

Comment:
If these are backticks, the `<` and `>` don't need to be escaped. They'll also be formatted as code:

```suggestion
Since is an async method that returns `Task`, a return keyword must not be followed by an object expression. Did you intend to return `Task<T>`?
```